vue/src/ImportVersion/ImportVersion.adapter.ts

```diff
@@ -0,0 +1,22 @@
+/*!
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+import { createAngularJsAdapter } from 'CoreHome';
+import ImportVersion from './ImportVersion.vue';
+
+export default createAngularJsAdapter({
+  component: ImportVersion,
+  scope: {
+    idContainer: {
+      angularJsBind: '=',
+    },
+    onSomething: {
+      angularJsBind: '&',
+    },
```

Comment:
What is that one used for?

Reply:
Nothing, this was part of a test, will remove it.

Reply:
```suggestion
```